deployment/openshift/web_dc.yaml

```diff
@@ -46,6 +46,7 @@ objects:
             "REACT_APP_DRAFT_ENABLED":"${REACT_APP_DRAFT_ENABLED}",
```

Comment:
I will create a ticket to remove these openshift templates

Reply:
okay